### DISCUSSION TITLE
OpenAI-like URL and model customizatiom feature

### DISCUSSION THREAD

**Discussion by XieyilinMain** [Taewan-P/gpt_mobile#97] [Ideas]
People from some countries cannot access OpenAI ChatGPT server, but they can access some other online LLMs, which also provide OpenAI-like URLs, like [https://dashscope.aliyuncs.com/compatible-mode](https://dashscope.aliyuncs.com/compatible-mode) .

This makes it compatible with 3rd-party LLMs.

**Comment by Taewan-P** [comment]
Duplicate of #62. Also your link is not accessible. (404)
FYI: https://github.com/Taewan-P/gpt_mobile/discussions/62#discussioncomment-10928255, https://github.com/Taewan-P/gpt_mobile/discussions/89#discussioncomment-11074735, https://github.com/Taewan-P/gpt_mobile/discussions/90#discussioncomment-11111238
